unison-cli/src/Unison/Codebase/TranscriptParser.hs

```diff
@@ -141,7 +141,7 @@ instance Show Stanza where
     API apiRequests ->
       "```api\n"
         <> ( apiRequests
-               & fmap (\(GetRequest txt) -> Text.unpack txt)
```

Comment:
This uni-pattern was actually a bug in the transcript parser before, 9.2.3 caught it 🎉